app/src/main/java/org/jellyfin/androidtv/details/trailers/YouTubeTrailerPresenter.kt

```diff
@@ -30,11 +39,29 @@ class YouTubeTrailerPresenter(private val context: Context, private val imageHei
 			contentText = context.getString(R.string.brand_youtube)
 		}
 
-		// TODO: Kick off loading of thumbnail here
-	}
+		if (!safeBrandingCompliance) {
+			Glide.with(context).load(trailer.thumbnailURL).listener(object : RequestListener<String, GlideDrawable> {
```

Comment:
IIRC you can use `.get()` instead of the listener when using a coroutine (GlobalScope.launch thingy...) If you want to catch the errors just use `try {} catch {}`